apps/src/lib/kits/maker/toolkit.js

```diff
@@ -109,6 +109,28 @@ export function connect({interpreter, onDisconnect}) {
     });
 }
 
+/**
+ * Called when the board disconnects
+ * Throw away reference to the currentBoard, so that next time we run
+ * we make a new board.
+ */
+function disconnect(onDisconnect) {
+  onDisconnect();
```

Comment:
Since this function doesn't interact with `onDisconnect` in any meaningful way, I'd rather remove it as a concern from the helper and just invoke it up a layer.  We might be able to give this helper a more meaningful name, too.

```js
currentBoard.once('disconnect', () => {
  onDisconnect();
  disconnect();
});

// ...

function disconnect() {
  if (/*...*/) {
```